components/pipeline-service/production/base/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
-  - https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=15f755cf4f4aaa575107f05ec3b44c92218802f1
+  - https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=589d6e4688334ed09eb4e10ddd5bdd7a01a5122b
```

Comment:
Why not update to the commit in staging 8313c38b61d684562cb93bd7d4757b08401d2c44 ?

Reply:
because my rebase attempt must have been faulty :-) 

I now see in main branch that it is now at 8313c38b61d684562cb93bd7d4757b08401d2c44 and not 589d6e4688334ed09eb4e10ddd5bdd7a01a5122b ... I will update thanks

Reply:
updated